stamply-backend/src/test/java/com/example/controller/business/BusinessControllerE2E.java

```diff
@@ -45,47 +46,50 @@ public class BusinessControllerE2E extends AbstractDatabaseTest {
     @Autowired
     protected BusinessRepository businessRepository;
 
+    @Autowired
+    TestRestTemplate restTemplate;
+
+    final HttpHeaders headers = new HttpHeaders();
+
+    @BeforeAll
+    static public void start() {
+        startMariaDBWithLoggingForClass(BusinessControllerE2E.class);
+    }
+
     @BeforeEach
-    public void setUp() {
-        mockMvc = standaloneSetup(businessController)
-                .apply(springSecurity(springSecurityFilterChain))
-                .build();
+    public void setUpHeaders() {
+        headers.add("Authorization", "Bearer " + createToken());
     }
 
     @Test
-    public void givenNonExistingBusinessWhenGotThenReturns404() throws Exception {
-        final var mvcResult = mockMvc.perform(
-                get("/api/business/")
-                        .accept(MediaType.APPLICATION_JSON)
-                        .contentType(MediaType.APPLICATION_JSON)
-                        .header("Authorization", "Bearer " + createToken())
-        ).andExpect(status().is(404)).andReturn();
+    public void givenNonExistingBusinessWhenGotThenReturns404() {
```

Comment:
does it work when ran by intellijIdea?
if it doesn't extend BaseSpringBootIT, then provideProperties won't be able to provide properties for mariaDB, therefore it won't be used.
Maven then is providing database.